feature/community/src/main/res/values/strings.xml

```diff
@@ -5,6 +5,5 @@
     <string name="communityUi_cardButton_communityWall">Community Wall</string>
```

Comment:
rename the string file, only use example:
<string name="title">Community Wall</string>
It is not necessary to specify which view they are from since it is a modularized app

Reply:
Same for the other strings, please remove the title and cardButton.

Reply:
the comments are not so clear, need more information, please elaborate over the thing that needs to be updated

Reply:
do you mean by any chance to adapt the nomenclature of each item value by removing the last part "_communityWall" from it ?  is this what you want to be done?

Reply:
Just remove the communityUI from all the strings

Reply:
remove "communityUi_", from all the strings

Reply:
IT was pushed yesterday at 4:00 the changes as I stated in Whatsapp coms